tally_ho/apps/tally/views/data/tally_list_view.py

```diff
@@ -29,6 +36,15 @@ def render_column(self, row, column):
         else:
             return super(TallyListDataView, self).render_column(row, column)
 
+    def filter_queryset(self, qs):
+        keyword = self.request.GET.get('search[value]', None)
+
+        if keyword:
+            qs = qs.filter(Q(name__contains=keyword) |
+                           Q(id__contains=keyword))
```

Comment:
Is the `id` field the primary key?

Reply:
Yes

Reply:
Do you think its weird to search that column this way?

Reply:
True, because it's not free text so it should be `iexact` instead since primary keys are unique but on second thought i think we should just remove this `id` filter because we only want a user to search by free text alone.

Reply:
Done